src/extensions/mod/core.py

```diff
@@ -162,10 +231,124 @@ async def ban_command(
                 reason=f"User banned by command. (Authorizing mod: {who}({who.id})"
             )
             self.bot.modlog.member_ban(mod=ctx.author, target=who, reason=reason)
-        else:
+        else:  # TODO: Verify: Is this guaranteed to convert to a member if the ID is a member? --Liz
```

Comment:
Thanks for this, I'll double-check the behavior with a stale member cache later.